docker-gitlabci/Dockerfile

```diff
@@ -7,7 +7,8 @@ RUN useradd -d /nonexistent -g nogroup -s /bin/false domjudge-run-0
 RUN useradd -d /nonexistent -g nogroup -s /bin/false domjudge-run-1
 RUN groupadd domjudge-run
 
-RUN apt-get update && apt-get install -y \
+RUN apt-add-repository ppa:pypy/ppa -y \
+  apt-get update && apt-get -y install \
```

Comment:
I rather test with the old version in the gitlab ci tests, we try to test DOMjudge here not if the ppa has connection issues or if they pushed a broken package.

I think we should always test with the oldest version someone could run, not with the version we normally use ourselves.